data/apps/scriptable.json

```diff
@@ -0,0 +1,56 @@
+{
+  "info": {
+    "identifier": "scriptable",
+    "name": "Scriptable",
+    "appstoreUrl": "https://itunes.apple.com/au/app/scriptable/id1405459188?mt=8"
+  },
+  "actions": [
+    {
+      "name": "Open Scriptable",
+      "description": "Opens Scriptable.",
+      "url": "scriptable://"
+    },
+    {
+      "name": "Add Script",
+      "description": "Add a new script.",
+      "url": "scriptable:///add"
```

Comment:
There is on `/` too many in here.

Also I don't understand this action. How is it that an `add` action does not have any parameters? Will this just create a completely empty script? This is where the link to some official docs would be great for more details.

Reply:
Wow thats super weird